server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -437,9 +479,9 @@ def parser_newscomponent(self, item, component_el):
             elements = keywords.findall('item_keyword')
 
             if elements is not None:
-                item['item_keywords'] = []
+                item['keywords'] = []
                 for element in elements:
-                    item['item_keywords'].append(element.text)
+                    item['keywords'].append(element.text)
```

Comment:
I would add an additional condition here: `for element in [e for e in elements if e.text]:` to avoid empty keyword entries.